src/batch_gemm.cc

```diff
@@ -120,16 +120,16 @@ void blas::batch::gemm(
     std::vector<int64_t>  const &m,
     std::vector<int64_t>  const &n,
     std::vector<int64_t>  const &k,
-    std::vector< std::complex<float>  >   const &alpha,
-    std::vector< std::complex<float>* >   const &Aarray, std::vector<int64_t> const &ldda,
-    std::vector< std::complex<float>* >   const &Barray, std::vector<int64_t> const &lddb,
-    std::vector< std::complex<float>  >   const &beta,
-    std::vector< std::complex<float>* >   const &Carray, std::vector<int64_t> const &lddc,
-    const size_t batch,                                  std::vector<int64_t>  &info )
+    std::vector< std::complex<float>  > const &alpha,
```

Comment:
Pedantic but why do we have space for `complex` vectors and not for `int64_t` vectors?
I guess it would be too much changes but I still ask in case it is something else.

Reply:
In the old days (pre C++11, I think)
`std::vector<std::complex<float>>`
was actually invalid syntax, because it thinks `>>` is the right shift operator. This isn't a concern anymore.

More generally, I find long template types with no spaces hard to read. We aren't 100% consistent, but generally when there are nested templates, I add spaces. In SLATE,
```
slate> git grep -P '\w+< \w+ >' | wc
    323    2174   24709
slate> git grep -P '\w+<\w+>' | wc
   9406   53312  783487
slate> git grep -P '\w+<std::complex<\w+>>' | wc
    198     786   16759
slate> git grep -P '\w+< std::complex<\w+> >' | wc
   1056    6016   83948
```